web/Views/Landlord/Dashboard.cshtml

```diff
@@ -7,62 +7,90 @@
-                <div class="col-5">
+                <div class="col">
                     <div class="row">
-                        <div class="col-3">
-                            <h1>
-                                <i class="bi bi-house-door"></i>
-                            </h1>
-                        </div>
-                        <div class="col-5">
-                            <h1>
-                                @Model.NumOfProperties
-                            </h1>
-                            <h6>Total Properties</h6>
-                        </div>
+                        <h6 class="font-body">Hello</h6>
+                        <h2 class="font-blue font-h1">@Model.CurrentLandlord.FirstName</h2>
+                        @if (Model.CurrentLandlord.NumOfProperties > 0)
+                        {
+                            <h6 class="font-body">View your properties</h6>
+                            <div>
+                                <a class="btn btn-primary btn-lg mt-3 align-middle" href='@Url.Action("ViewProperties", "Landlord", new { id = Model.CurrentLandlord.LandlordId })'>View Properties</a>
+                            </div>
+                        }
+                        else
+                        {
+                            <h6 class="font-body">Start adding your properties to the portal, to get matches</h6>
+                            <div>
+                                <a class="btn btn-primary btn-lg mt-3 align-middle" href='@Url.Action("AddNewProperty_Begin", "Property", null)'>Add Properties</a>
+                            </div>
+                        }
+                        @if (!Model.CurrentLandlord.CharterApproved)
+                        {
+                            <p class="font-subtitle font-orange mt-1">
+                                *your properties won't go live until the ChangeAhead charter signup has been approved by the ChangeAhead admins. You can still add and update draft properties before then.
+                            </p>
+                        }
                     </div>
                 </div>
             </div>
         </div>
-        <div class="col-4">
-            <h6>My Matches</h6>
+        <div class="col">
+            <h6 class="font-h3">My Stats</h6>
+            @if (Model.CurrentLandlord.CharterApproved)
+            {
+                await Html.RenderPartialAsync("_DashboardStatsBoxPartial", new StatBoxModel
+                {
+                    IconName = "bi-check",
+                    Stat = "Approved",
+                    StatDescription = "Charter Approval State"
+                });
+            }
+            else
+            {
+                await Html.RenderPartialAsync("_DashboardStatsBoxPartial", new StatBoxModel
+                {
+                    IconName = "bi-x",
+                    Stat = "Not Approved",
+                    StatDescription = "Charter Approval State"
+                });
```

Comment:
This could be combined slightly and would remove the need of the if
```suggestion
                await Html.RenderPartialAsync("_DashboardStatsBoxPartial", new StatBoxModel
                {
                    IconName = Model.CurrentLandlord.CharterApproved ? "bi-check" : "bi-x",
                    Stat = Model.CurrentLandlord.CharterApproved ? "Approved" : "Not Approved",
                    StatDescription = "Charter Approval State"
                });
```